packages/volto/news/5645.bugfix

```diff
@@ -0,0 +1 @@
+Added a simple condition to not execute the "closeMenu" in handleClickOutside function when click on the "icon" because it will execute by toggleMenu function. @vikash-prem123
```

Comment:
News items usually should make sense to both end users and developers. Implementation details are obvious when a developer reads the diff.

```suggestion
The user personal tools menu now closes when clicked a second time or when clicking outside of the component. @vikash-prem123
```